app/validators/ip_regexp_validator.rb

```diff
@@ -1,9 +1,5 @@
 class IpRegexpValidator < ActiveModel::EachValidator
```

Comment:
It might be a good idea to simply rename this to RegexpValidator now, we may find other things containing regexes that could be validated in the same way as there's nothing IP-specific in it any more.